Snowflake.Data.Tests/SFMultiStatementsIT.cs

```diff
@@ -302,7 +302,7 @@ public void testWithAllQueryTypes()
                 {
                     cmd.CommandText = "select 1;" +
                                       "create or replace temporary table t1(c1 varchar);" +
-                                      "explain using tabular select * from t1;" +
+                                      "explain using text select * from t1;" +
```

Comment:
Why change the SQL?

Reply:
On our dev envs we get: SQL compilation error: error line 1 at position 4 Invalid explain plan format 'tabular'.
Since that is not the main purpose of the test (we test here multi line statements) I've change it to 'text'.
One more info - setting FORCE_USING_EXTERNAL_EXPLAIN = true fixes the problem with 'tabular' on local env.